### ISSUE TITLE
Bad hex code when using 212121

### ISSUE THREAD

**Issue by paw-lu** [joowani/colorpedia#5]
Cool tool!
Thank you so much for sharing it!

Entity  | Version
--- | ---
colorpedia | 1.1.0
Python | 3.9.1
 

When I run

```sh
% color hex 212121
Bad hex code (expecting a string matching ^(?:[0-9a-fA-F]{3}){1,2}$)
```

#212121 seems to be [a valid hex code](https://www.google.com/search?q=%23212121&oq=%23&aqs=chrome.0.69i59j69i57j69i59l2j46i433j0i131i395i433j0i3i395j0i395j46i175i199i395j69i59.766j1j4&sourceid=chrome&ie=UTF-8).

**Comment by joowani** [comment]
Hi @paw-lu,

Thanks for the bug report! I'll release a fix soon.

**Comment by joowani** [comment]
I just released 1.2.0 with the fix. Please run `pip install colorpedia --upgrade` to upgrade. Feel free to reopen if you still have issues. Thanks!

**Comment by micahyoung324** [comment]
Hello, I encountered the same issue when I entered `color hex 000000`.

**Comment by paw-lu** [comment]
I can confirm `212121` works, but `000000` throws a `Bad hex code` on `1.2.1`.

**Comment by joowani** [comment]
Thanks @mxw7265 and @paw-lu. I've released `1.2.2` with the fix for `000000`.

**Comment by ijomeli** [comment]
all the above cases now working for me, but now getting the  `Bad hex code (expecting a string matching ^(?:[0-9a-fA-F]{3}){1,2}$)` message for `191E32` on version `1.2.2`.
weirdly though, replacing the `E` with any other hex character works fine, so `191F32` or `191932` both run as expected. it's just `191E32` which has a problem.

**Comment by joowani** [comment]
Hi @lamados,

Thanks for the bug report. All the problems in this ticket have been from [python-fire](https://github.com/google/python-fire) (which Colorpedia uses under the hood). Fire is interpreting 000000, 212121 and 191E32 as numbers. There doesn't seem to be a way to force it to interpret them as strings. I'm planning to switch to Python's builtin `argparse` in the next release. Until then, you can do this:

```shell
color hex '"191E32"'
```

**Comment by joowani** [comment]
Hi again @lamados,

Actually, I found a workaround so I'll first add that in the next release soon.

**Comment by joowani** [comment]
I published a new release 1.2.3 with the bugfix.
